src/metabase/api/transform.clj

```diff
@@ -0,0 +1,15 @@
+(ns metabase.api.transform
+  (:require [compojure.core :refer [GET]]
+            [medley.core :as m]
+            [metabase.api.common :as api]
+            [metabase.transforms
+             [core :as transform]
+             [specs :as transform.specs]]))
+
+(api/defendpoint GET "/:db-id/:schema/:transform-name"
```

Comment:
Can we get a docstring for the endpoint

Reply:
Also we need some tests for this API endpoint

Reply:
This is only provisional so we can play with it. It'll get removed once we have a solid grasp on how to interface with transforms.